dockerfile

```diff
@@ -1,26 +1,56 @@
-# ---- Base Node ----
-FROM node:alpine AS base
+FROM node:18-alpine AS base
```

Comment:
This locks us in to node 18, but honestly.. we probably won't need any cutting edge features as of yet.
Add to todo ;P

Reply:
does the latest alpine not work?